### DISCUSSION TITLE
Routing problem ... only in Cruiser

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#561] [Q&A] [answered]
Until 3 weeks ago there was a month-long road closure due to construction work at this location. This has now been removed and someone has corrected it in OSM. Unfortunately, this correction means that Cruiser still does not want to use the bridge. BRouter, GMaps and Kurviger have no problem with it. Can someone please take a look at this? 
This is the location:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/1d5b4fa5-3417-45db-bdb2-9f5805830b7d"/>
the corresponding gpx-file: [RoutingProblem.gpx.txt](https://github.com/user-attachments/files/15509788/RoutingProblem.gpx.txt)
and the BRouter website: https://brouter.de/brouter-web/#map=15/48.1682/11.7433/osm-mapnik-german_style&lonlats=11.746928,48.170919;11.733148,48.163902&profile=car-fast

Thanks in advance!

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the report.

Do you mean with BRouter (offline)?
Where exactly is the problem?

I did a routing with BRouter (offline) with updated routing data and driving profile with fastest and default options
using the start / end of the gpx rte and it works (both directions).

<img src="https://github.com/devemux86/cruiser/assets/3484020/23a785c3-58ba-4103-bc50-864a3f2082f5" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/7839b23e-7c78-4ba8-b0e4-7c25c3c0d744" width="50%"/>

**Reply by WolfgangZ42** [reply, marked as answer]
Sorry - I didn't update the BRouter tiles for a while. 🥴 
**Now it works!** I should have thought about before writing ...
